src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -18,6 +18,9 @@ public interface INuGetProjectManagerService : IDisposable
         ValueTask<IReadOnlyCollection<IPackageReferenceContextInfo>> GetInstalledPackagesAsync(
             IReadOnlyCollection<string> projectIds,
             CancellationToken cancellationToken);
+        ValueTask<IReadOnlyCollection<IPackageReferenceContextInfo>> GetTransitivePackagesAsync(
```

Comment:
Ah, I understand now why it was necessary to call `.AsTask()` earlier. But this shows me it's even more important for there to be a single method to get both direct and transitive packages in one call. Doing them separately will have big perf impact when connected to CodeSpaces. Ideally the entire Project list gets passed, and all the results returned, in a single call. Maybe we should talk to @dtivel about this.

Reply:
Changed GetTransitivePackagesAsync to GetAllPackagesAsync to get both installed and transitive packages at the same time.